creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.ts

```diff
@@ -425,11 +434,13 @@ const _getUserPrimaryClockValues = async (wallets: string[]) => {
 }
 
 /**
+ * @deprecated - For backwards compat only. Use _additionalSyncIsRequired to poll by UUID instead of clock value.
```

Comment:
idrk best process here but we historically have deprecated back-compat code littered around base
i started just adding a TODO with a target service version and a corresponding linear card with a due date to remove
if you think of better approach lmk

Reply:
sgtm ✅